vmngclient/api/dashboard_api.py

```diff
@@ -75,3 +75,79 @@ def get_control_statuses_count(self) -> DataSequence[Count]:
             status.status_list = DataSequence(Count, [create_dataclass(Count, item) for item in status.status_list])
 
         return statuses
+
+    def get_bfd_connectivity_count(self) -> DataSequence[Count]:
+        """
+        Get information about bfd connectivity count.
+
+        Returns:
+            DataSequance of Count dataclass with bfd connectivity
+        """
+        response = self.session.get("/dataservice/device/bfd/sites/summary")
+
+        bfd_response = response.dataseq(Count)
+        for item in bfd_response:
+            item.status_list = DataSequence(
+                Count, [create_dataclass(Count, bfd_connection) for bfd_connection in item.status_list]  # type:ignore
```

Comment:
We should cover that inside `dataseq` method or re-define our deserialization

Reply:
#210